vip-cache-manager/vip-cache-manager.php

```diff
@@ -26,45 +36,43 @@ function __construct() {
 		add_action( 'init', array( $this, 'init' ) );
 	}
 
-	function init() {
+	public function init() {
 		if ( is_super_admin() && isset( $_GET['cm_purge_all'] ) && check_admin_referer( 'manual_purge' ) ) {
```

Comment:
For single sites, I think `manage_options` makes sense. It's probably also OK for sub-domain multisites. It may make sense to require super_admin on sub-directory multisites, at least on the main site, as that will clear cache for the entire network.